exe/Daemon.hs

```diff
@@ -202,25 +206,25 @@ send id (Api.SendRequest expressions) = do
       let isResponse = \case
             New InputsApplied{ nonce = Just nonce' } | nonce' == nonce -> True
             _ -> False
-      asyncMsg <- liftIO $ async $ subscribeOne sub ackWaitTime isResponse (logNonDecodableMsg id)
-      ipfs id $ publish id (Submit SubmitInputs{..})
+      asyncMsg <- machineIpfs id $ async $ subscribeOne sub ackWaitTime isResponse (logNonDecodableMsg id)
```

Comment:
Do we need `machineIpfs id` here? It might be confusing because this will never throw an `IpfsException`. `liftIO` should suffice